unit_tests/test_data/test_scylla_yaml_builders/PR-provision-test.result.json

```diff
@@ -14,5 +14,9 @@
     }
   ],
   "experimental": true,
-  "prometheus_address": "0.0.0.0"
+  "prometheus_address": "0.0.0.0",
+  "alternator_enforce_authorization": false,
+  "auto_bootstrap": true,
+  "enable_ipv6_dns_lookup": false,
+  "hinted_handoff_enabled": "enabled"
```

Comment:
@soyacz why do we have those 4 extra ones ? where they are coming from ? the original yaml had them and then were equal to the defaults ?

Reply:
I think it comes from `ScyllaYamlClusterAttrBuilder`, previously they were defaults -> skipped from yaml.